### ISSUE TITLE
EditorState "dragged" event not being set

### ISSUE THREAD

**Issue by bozz** [prevwong/craft.js#321]
**Describe the bug**
When a node is being dragged, the EditorState `dragged` event is not being set to NodeId - it always remains `null`.

**To Reproduce**
Adding this to the `useEditor` hook - the `dragged` always remains `null`. Tested this within the Basic example from the craftjs repo (within components/Toolbox.js). The other two events are being set correctly.

```js
const { connectors } = useEditor((state) => {
    console.log("Dragged: ", state.events.dragged);
    console.log("Selected: ", state.events.selected);
    console.log("Hovered: ", state.events.hovered);
 });
```

**Expected behavior**
I expect the `dragged` event to be correctly set to the dragged node's NodeId within `useEditor`.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| craft.js         | 0.1.0-beta.20
| React            | 17.0.2
| Browser          | Chrome, Firefox
| Operating System | MacOS

**Comment by scottallanaz** [comment]
Hello, has anyone found a resolution to this?

Thanks

**Comment by prevwong** [comment]
Fixed in v0.1.1 💯
